synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -487,6 +487,37 @@ def recur_animate(self, anim_type):
                 self.expression = ret
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "cos":
+                self.subparams["cos"].extract_subparams()
+                angle, eff_1 = self.subparams["cos"].subparams["angle"].recur_animate("region_angle")
+                amp, eff_2 = self.subparams["cos"].subparams["amp"].recur_animate("real")
+                self.expression_controllers.extend(eff_1)
+                self.expression_controllers.extend(eff_2)
+                
+                if self.dimension == 2:
+                    ret = "mul(Math.cos(degreesToRadians({angle})), [{amp}, {amp}])"
```

Comment:
For simple circle, we need to give diameter  to the lottie file, but here we are giving radius as input.
This needs to be changed "only for support of simple circle".
Somehow it should be given {amp}*2 only for simple circle

Reply:
This might fail under some circumstances like when there is a sine function being called recursively inside another sine function.

Reply:
Yup, you are right. Using {amp}*2 will fail when called in recursively. Hence I purpose that you modify the original animation(I mean modify the xml itselves, just like [here](https://github.com/synfig/synfig/blob/77164e8371269afb0490287f200f6c35fe067e88/synfig-studio/plugins/lottie-exporter/synfig/animation.py#L17)) and multiply the circle_radius by 2 at the root level

Reply:
Resolved now. :D